postman/Users.postman_collection.json

```diff
@@ -1,289 +1,289 @@
-							"value": "Sun, 09 Aug 2020 11:22:58 GMT"
-						}
-					],
-					"cookie": [],
-					"body": "{\n    \"currentPassword\": \"strongPassword1_\",\n    \"newPassword\": \"strongPassword2_\",\n    \"repeatPassword\": \"strongPassword2_\"\n}"
-				}
-			]
-		}
-	],
-	"event": [
-		{
-			"listen": "prerequest",
-			"script": {
-				"id": "37121c31-0101-4c58-9d9b-f2de4c702feb",
-				"type": "text/javascript",
-				"exec": [
-					""
-				]
-			}
-		},
-		{
-			"listen": "test",
-			"script": {
-				"id": "831cc249-48bc-4ed5-8b6f-af269297b121",
-				"type": "text/javascript",
-				"exec": [
-					""
-				]
-			}
-		}
-	],
-	"protocolProfileBehavior": {}
+  "info": {
+    "_postman_id": "f5d8e17f-cbb6-4568-8c9c-c572d5dc90a9",
+    "name": "Users",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+  },
+  "item": [
+    {
+      "name": "Find user by email",
+      "request": {
+        "method": "GET",
+        "header": [
+          {
+            "key": "Authorization",
+            "value": "{{auth_token}}",
+            "type": "text"
+          }
+        ],
+        "url": {
+          "raw": "{{base_url}}/users?email=example@mail.ru",
+          "host": [
+            "{{base_url}}"
+          ],
+          "path": [
+            "users"
+          ],
+          "query": [
+            {
+              "key": "email",
+              "value": "example@mail.ru"
+            }
+          ]
+        }
+      },
+      "response": [
+        {
+          "name": "Find user by email example",
+          "originalRequest": {
+            "method": "GET",
+            "header": [
+              {
+                "key": "Authorization",
+                "value": "{{auth_token}}",
+                "type": "text"
+              }
+            ],
+            "url": {
+              "raw": "{{users_url}}?email=example@mail.ru",
+              "host": [
+                "{{users_url}}"
+              ],
+              "query": [
+                {
+                  "key": "email",
+                  "value": "example@mail.ru"
+                }
+              ]
+            }
+          },
+          "status": "OK",
+          "code": 200,
+          "_postman_previewlanguage": "json",
+          "header": [
+            {
+              "key": "Server",
+              "value": "akka-http/10.1.10"
+            },
+            {
+              "key": "Date",
+              "value": "Sun, 09 Aug 2020 11:00:07 GMT"
+            },
+            {
+              "key": "Content-Type",
+              "value": "application/json"
+            },
+            {
+              "key": "Content-Length",
+              "value": "292"
+            }
+          ],
+          "cookie": [],
+          "body": "[\n    {\n        \"userId\": \"26774154-8946-4764-b8ab-37dfeccd3453\",\n        \"email\": \"example@mail.ru\",\n        \"firstName\": \"Ivan\",\n        \"lastName\": \"Ivanov\",\n        \"active\": true\n    }\n]"
```

Comment:
It is really hard to find the difference here :) Is it only that line?

Reply:
@Nikolai-Isaev yes, I have changed it for correct display of "Find user by email example" in postman.